app/helpers/templates_helper.rb

```diff
@@ -81,7 +81,15 @@ def template_input_f(f, options = {})
       elsif input_type == 'search'
         input_type = 'autocomplete'
         resource_type = input.resource_type&.tableize
-        options.merge!(resource_type: resource_type, use_key_shortcuts: false, url: search_path(resource_type))
+        options[:data] = {
+          autocomplete: {
+            searchQuery: options[:search_query] || f.object&.value || '',
+            controller: options[:path] || auto_complete_controller_name,
+            disabled: options[:disabled] || false,
+            url: search_path(resource_type),
+          },
+        }
+        options[:onSearch] = nil
```

Comment:
Can we use `autocomplete_f` in this case?
it creates the same `data` object with the same defaults:
https://github.com/theforeman/foreman/blob/97282e16bf49a14510e7b49badc2c99d9b30a48b/app/helpers/form_helper.rb#L222

Reply:
It creates it from different options for the `searchQuery` and `url` keys so I think we should keep them seperate